modules/openapi-generator/src/test/resources/3_0/features/dynamic-servers.yaml

```diff
@@ -0,0 +1,92 @@
+openapi: 3.0.0
+info:
+  description: This specification shows how to use dynamic servers.
```

Comment:
@jimschubert I have tried to create a minimal specification on which we can showcase the dynamic server configuration to avoid modifications of default "Petstore" example.